install.sh

```diff
@@ -0,0 +1,502 @@
+#!/usr/bin/env bash
+
+# The MIT License (MIT)
+
+# Copyright (c) 2010 Tim Caswell
```

Comment:
Probably want to update this

Reply:
updated the license here https://github.com/temporalio/cli/pull/50